src/redux/constants/actionTypes.js

```diff
@@ -12,13 +12,17 @@ export const OPEN_OFFER_MODAL = "OPEN_OFFER_MODAL";
 export const OPEN_POSTING_MODAL = "OPEN_POSTING_MODAL";
 
 /**
- * Posting Action Types
+ * Offering Action Types
```

Comment:
This file will need to be cleaned up/reordered at eventually. Holding off for now because there have been too many changes and potential for merge conflicts

Reply:
sounds good. Should I revert the change for now then? Or is it okay to keep the changed comment?

Reply:
it's probably fine to leave it